cmd/eval_test.go

```diff
@@ -332,6 +340,15 @@ func testEvalWithSchemaFile(t *testing.T, input string, query string, schema str
 			t.Fatal(err)
 		}
 
+		if expTypeErr {
+			if len(output.Errors) != 1 || output.Errors[0].Code != "rego_type_error" {
+				err = fmt.Errorf("expected type conflict")
+			} else {
+				err = nil
```

Comment:
[nit] I don't think we need this
```suggestion
```

Reply:
Actually, `err` could contain an expected error from the call to `eval`. But I think readability is improved if I remove this and don't reuse `err` in the above call.